packages/next/taskfile.js

```diff
@@ -72,8 +72,7 @@ const externals = {
   chokidar: 'chokidar',
   'jest-worker': 'jest-worker',
 
-  'terser-webpack-plugin':
-    'next/dist/build/webpack/plugins/terser-webpack-plugin/src',
+  'terser-webpack-plugin': 'terser-webpack-plugin',
```

Comment:
Required for `bundle5.js` to work. Please see https://github.com/vercel/next.js/pull/57656#issuecomment-1954371776

Reply:
I'm planning to leave this and bundle5 for now, since it did fix error.